content/en/docs/reference/kube-scheduler/profiles.md

```diff
@@ -0,0 +1,174 @@
+  volumes.
+  Extension points: *Filter*.
+- `VolumeRestrictions`: Checks that volumes mounted in the node satisfy
+  restrictions which are specific to the volume provider.
+  Extension points: *Filter*.
+- `VolumeZone`: Checks that volumes requested satisfy any zone requirements they
+  might have.
+  Extension points: *Filter*.
+- `NodeVolumeLimits`: Checks that CSI volume limits can be satisfied for the
+  node.
+  Extension points: *Filter*.
+- `EBSLimits`: Checks that EBS volume limits can be satisfied for the node.
+  Extension points: *Filter*.
+- `GCEPDLimits`: Checks that GCP-PD volume limits can be satisfied for the node.
+  Extension points: *Filter*.
+- `AzureDiskLimits`: Checks that Azure disk volume limits can be satisfied for
+  the node.
+  Extension points: *Filter*.
+- `InterPodAffinity`: Implements
+  [inter Pod affininity and antiaffinity](/docs/concepts/configuration/assign-pod-node/#inter-pod-affinity-and-anti-affinity).
+  Extension points: *PreFilter*, *Filter*, *PreScore*, *Score*.
+- `PrioritySort`: Provides the default priority based sorting.
+  Extension points: *QueueSort*.
+- `DefaultBinder`: Provides the default binding mechanism.
+  Extension points: *Bind*.
+  
+You can also enable the following plugins, through the component config APIs,
+that are not enabled by default:
+
+- `NodeResourcesMostAllocated`: Favors nodes that have a high allocation of
+  resources.
+  Extension points: *Score*.
+- `RequestedToCapacityRatio`: Favor nodes according to a configured function of
+  the allocated resources.
+  Extension points: *Score*.
+- `NodeResourceLimits`: Favors nodes that satisfy the Pod resource limits.
+  Extension points: *PreScore*, *Score*.
+- `CinderVolume`: Checks that Cinder volume limits can be satisfied for the
+  node.
+  Extension points: *Filter*.
+- `NodeLabel`: Filters and/or scores a node according to the configured labels.
+  Extension points: *Filter*, *Score*.
+- `ServiceAffinity`: Checks that Pods that belong to a service fit in a set of
+  nodes defined by configured labels. It also favors spreading the Pods
+  belonging to a service across nodes.
+  Extension points: *PreFilter*, *Filter*, *Score*.
+  
+## Multiple profiles
+
+When using the component config API v1alpha2, a scheduler can be configured to
+run more than one profile. Each profile has an associated scheduler name.
+Pods that want to be scheduled according to a specific profile can include
+the corresponding scheduler name in its `.spec.schedulerName`.
+
+By default, one profile with the scheduler name `default-scheduler` is created.
+This profile includes the default plugins described above. When declaring more
+than one profile, a unique scheduler name for each of them is required.
+
+If a Pod doesn't specify a scheduler name, kube-apiserver will set it to
+`default-scheduler`. Therefore, a profile with this scheduler name should exist
+to get those pods scheduled.
```

Comment:
This could go in the concept page I think, perhaps the notes that follow it as well?

It's OK to repeat the notes on both pages.

Reply:
I prefer to just keep this here, as it is all part of the same API.